helm-charts/azure-api-management-gateway/README.md

```diff
@@ -121,6 +121,13 @@ their default values.
 | `podSecurityContext`                                   | Privilege and access control settings for the  pod                                                                                                                                                                                                    | `{}`                                                        |
+| `backup.enabled`                                   | If enabled will store a backup copy of the latests downloaded configuration on a storage volume                                                                                                                                                                                                 | `false`                                                        |
+| `backup.existingPVCName`                                   | Use an existing Persistent Volume Claim instead of creating one. backup.createPVC needs to be false                                                                                                                                                                                                | `""`                                                        |
+| `backup.createPVC`                                   | Create a Persistent Volume Claim with values specified in backup.PVC*                                                                                                                                                                                                     | `true`                                                        |
+| `backup.PVC.storageClassName`                                   | storageClassName to be set on the Persistent Volume Claim. `null` means no storageClassName specified and will use the platform default, `""` means no storageClassName specified and none will be used.                                                                                                                                                                                                    | `null`                                                        |
+| `backup.PVC.accessMode`                                   | accessMode for the Persistent Volume Claim  pod                                                                                                                                                                                                    | `ReadWriteMany`                                                        |
+| `backup.PVC.size`                                   | Size of the Persistent Volume to be created                                                                                                                                                                                                    | `50Mi`                                                        |
```

Comment:
I think it would be good if they are all aligned, for example `*.backup.persistenVolumeClaim.create`
```suggestion
| `gateway.configuration.backup.persistenVolumeClaim.existingName`                                   | Use an existing Persistent Volume Claim (PVC) instead of creating one. backup.createPVC needs to be false                                                                                                                                                                                                | `""`                                                        |
| `gateway.configuration.backup.persistenVolumeClaim.create`                                   | Create a Persistent Volume Claim (PVC) with values specified in backup.PVC*                                                                                                                                                                                                     | `true`                                                        |
| `gateway.configuration.backup.persistenVolumeClaim.storageClassName`                                   | storageClassName to be set on the Persistent Volume Claim (PVC). `null` means no storageClassName specified and will use the platform default, `""` means no storageClassName specified and none will be used.                                                                                                                                                                                                    | ``                                                        |
| `gateway.configuration.backup.persistenVolumeClaim.accessMode`                                   | Access mode for the Persistent Volume Claim (PVC) pod                                                                                                                                                                                                    | `ReadWriteMany`                                                        |
| `gateway.configuration.backup.persistenVolumeClaim.size`                                   | Size of the Persistent Volume Claim (PVC) to be created                                                                                                                                                                                                    | `50Mi`                                                        |
```

The names might feel long but when reading configs this would help or if people are not sure what a PVC means.

Reply:
This should be resolved in the latest commit

Reply:
Thanks!